patterns/1-initial/capacity-for-contributions.md

```diff
@@ -0,0 +1,61 @@
+## Title
+
+Managing capacity for reviewing contributions
+
+## Patlet
+
+Reviewing InnerSource contributions takes time and effort. This should be reflected in capacity planning, especially for larger contributions. Expectations and available capacity should be transparent so that contributors understand when their contributions will be reviewed and released.
+
+## Problem
+
+Large InnerSource contributions are causing delays to other work in the team and/or contributions are taking longer to be released than expected. Reviewing contributions may be significant invisible work, not tracked in a team's agile development process.
```

Comment:
Agile is uppercase just like InnerSource is

Reply:
https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/692/commits/17b0f76dca1aef13048d6b1ab576ad1fc877b985